src/overlays/actors/ovl_En_Hgo/z_en_hgo.c

```diff
@@ -337,7 +336,7 @@ void EnHgo_Update(Actor* thisx, PlayState* play) {
     this->actionFunc(this, play);
     SkelAnime_Update(&this->skelAnime);
     if (func_80BD0898(this, play)) {
-        Actor_TrackNone(&this->unk_300, &this->unk_306);
+        Actor_TrackNone(&this->headRot, &this->torsoRot);
```

Comment:
Strictly speaking it is used, then?

Reply:
Technically yes, but it is more those functions just fill that value which the actor never uses.